.github/CODEOWNERS

```diff
@@ -1113,11 +1113,16 @@ x-pack/plugins/cloud_integrations/cloud_full_story/server/config.ts @elastic/kib
 /x-pack/plugins/stack_connectors/common/gen_ai @elastic/security-threat-hunting-explore
```

Comment:
Wouldn't be it a bit cleaner if do not mix domain names with model/api_routes on the same level.
Instead creating folder `domains` or so on and put domain folders there. This way we don't mix "technical" folders(models, apis) with names of domains

before
<img width="241" alt="Screenshot 2023-07-14 at 10 19 27" src="https://github.com/elastic/kibana/assets/92328789/7be31365-235f-4ea5-a613-8d84fd38c40b">

after
<img width="416" alt="Screenshot 2023-07-14 at 10 21 52" src="https://github.com/elastic/kibana/assets/92328789/b3375ccc-c610-4954-b92c-eeabee8a7425">

Or these folders on level with `model` are api routes? How to distinguish domains and routes, when looking at folder tree?

according to

> security_solution/
| - common/
| | - api/
| | | - detection_engine/
| | | | - api_routes/
| | | | - model/
| | | | - sub_domains/
| | | | | - api_routes/
| | | - other_domains/

Reply:
In a way I saw each individual api route its own small domain, which is how I justified not distinguishing between domains and API routes in the folder structure. In my mind the API route is just a special case of domain that only has a single API route in the whole domain, so in that case the API route files are in the folder as opposed to having more sub-folders with sub-domains.

> How to distinguish domains and routes, when looking at folder tree?

Agreed that it could be useful to have a standard way to distinguish between API route folders and domains. Right now API routes generally have some kind of verb in the name (get, find, install, review, etc), but perhaps appending `_route` to the API route folders in all cases would make it clearer?

> Or these folders on level with model are api routes?

Sibling folders to `model` could be API routes or sub-domains - in the "API route is a special case of domain" paradigm there's not much of a distinction. Basically every folder is a domain except folders named `model`, which is reserved for non-API route shared components.

Reply:
> Agreed that it could be useful to have a standard way to distinguish between API route folders and domains. Right now API routes generally have some kind of verb in the name (get, find, install, review, etc), but perhaps appending _route to the API route folders in all cases would make it clearer?

I personally would prefer, to have them in separate folder `routes`, which is on the same level with `model`. But I am fine with that approach too.